packages/eslint-plugin/ROADMAP.md

```diff
@@ -177,7 +177,7 @@
 | [`prefer-while`]                    | 🛑  | N/A                                                                                 |
 | [`quotemark`]                       | 🌟  | [`quotes`][quotes]                                                                  |
 | [`return-undefined`]                | 🛑  | N/A                                                                                 |
-| [`semicolon`]                       | 🌟  | [`semi`][semi]                                                                      |
+| [`semicolon`]                       | 🌓  | [`@typescript-eslint/semi`]                                                         |
```

Comment:
you have to add the reference to the list at the bottom of the file, or else it won't create a link.
![image](https://user-images.githubusercontent.com/7462525/55975153-2e897980-5c3e-11e9-80ce-5a92b3f79b6e.png)